recipes/hlama/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: hlama
+  version: '0.3'
+
+build:
+  number: 0
+  skip: True # [py27 or py35 or osx]
```

Comment:
Ah, HLA-MA only works for Python 3 because it depends on Snakemake.

No worries, no hurry. Thanks! 🙇